templates/base_index.html

```diff
@@ -1409,56 +1409,83 @@ <h2>
     * Choose a takeover
     * ===
     *
-    * From the list of provided takeovers in the #takeovers template,
+    * From the list of provided takeovers from /.takeovers.json,
     * choose one (that matches the client's language), and replace the
     * base template with it.
     */
 
-    var baseTakeover = document.getElementById('takeover');  // The base takeover element
-    var takeovers = document.getElementById('takeovers');  // The list of current takeovers to choose from
-
-    // For browsers that support <template> (all but IE), grab the template's ".content"
-    takeovers = 'content' in takeovers ? takeovers.content: takeovers;
+    fetch("/takeovers.json").then(response => response.json()).then(takeovers => {
```

Comment:
Does ubuntu.com have `fetch` polyfill for older browsers or do we rely on native implementation - quite fresh browsers are required to use it natively: https://caniuse.com/?search=fetch

Reply:
Rewritten with XHR

Reply:
There is a well grounded fetch implementation. If you call APIs a lot probably worth adding as dependency: https://github.com/github/fetch